docs/meshstack.aws.aad-as-idp-for-aws.md

```diff
@@ -0,0 +1,85 @@
+---
```

Comment:
d: Since this is a new page, it should be added to sidebars.json file as well otherwise it won't show in the docs unless explicitly defined in the url

Reply:
Thanks, i fixed it. Is the title of the page okay? Its hard to find a short title that looks good in the navigation

Reply:
Would it be more suitable is "AWS SSO integration"? @sttomm maybe you have more insights on this?

Reply:
As mentioned by stefan in another comment, i change the name